mne/decoding/transformer.py

```diff
@@ -715,3 +719,157 @@ def transform(self, X):
         X = self.estimator.transform(X)
         X = np.reshape(X.T, [-1, n_epochs, n_times]).transpose([1, 0, 2])
         return X
+
+
+class TemporalFilter(TransformerMixin):
+    """Estimator to filter data array along the last dimension, having at
+    least 2 dimensions. Can be applied on epochs, evoked data.
```

Comment:
Remove 'Can be applied on epochs and evoked data', it suggests that we can pass the corresponding MNE objects